docs/concepts/glossary.md

```diff
@@ -198,7 +198,7 @@ A Cryptographic Hash is a function that takes some arbitrary input (content) and
 
 ### Hole punching
 
-A generic term for NAT traversal when port forwarding is not possible. [See DCUtR](#dcutr)
+A  technique for NAT / firewall traversal that relies on coordinated simultaneous connections.  Used when port forwarding is not possible. [See DCUtR](#dcutr)
```

Comment:
```suggestion
A  technique for [NAT](#nat) or firewall traversal that relies on coordinated simultaneous connections.  Used when port forwarding is not possible. [See DCUtR](#dcutr)
```

Oh, turns out we don't have anything for NAT.

Reply:
Added in https://github.com/ipfs/ipfs-docs/pull/954/commits/6a789d135f4d676f2d818446f67d5e74f2ecd2c0